app/controllers/account/admin/billing_contracts_controller.rb

```diff
@@ -47,7 +47,7 @@ def destroy
   private
 
   def billing_contract_params
-    params.require(:billing_contract).permit(:contract_num, :is_active)
+    params.require(:billing_contract).permit(:contract_num, :is_active, :user_id, :company_id)
```

Comment:
what for is user_id?

Reply:
they are in the table of billing_contracts, so later we will need them here. I may delete them if you want...

Reply:
> they are in the table of billing_contracts, so later we will need them here. I may delete them if you want...

don't add those "later" things, it's better to add them at the time they should be used